docs/whats-new/2020-06.md

```diff
@@ -0,0 +1,136 @@
+  - Add System.Net APIs (2 of 3)
+
+## F# language
+
+### Updated articles
+
+- [Keyword Reference](/dotnet/fsharp/language-reference/keyword-reference)
+  - Fix default checksum algorithm
+
+## ML.NET
+
+### Updated articles
+
+- [The ML.NET CLI command reference](/dotnet/machine-learning/reference/ml-net-cli-reference)
+  - Update mlnet cli docs
+
+## .NET for Apache Spark
+
+### New articles
+
+- [Use broadcast variables in .NET for Apache Spark](/dotnet/spark/how-to-guides/broadcast-guide) - Added docs for broadcast and UDF guide
+- [Create user-defined functions (UDF) in .NET for Apache Spark](/dotnet/spark/how-to-guides/udf-guide) - Added docs for broadcast and UDF guide
+
+## .NET
+
+### New articles
+
+- [Implement a DisposeAsync method](/dotnet/standard/garbage-collection/implementing-disposeasync) - Implement the async dispose pattern
+
+## Community contributors
+
+The following people contributed to the .NET docs in June 2020. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [NextTurn](https://github.com/NextTurn) - Next Turn (52)
+- [DennisLee-DennisLee](https://github.com/DennisLee-DennisLee) - Dennis Lee (32)
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (14)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (8)
+- [MarcoRossignoli](https://github.com/MarcoRossignoli) - Marco Rossignoli (4)
+- [nschonni](https://github.com/nschonni) - Nick Schonning (3)
+- [alfredmyers](https://github.com/alfredmyers) - Alfred Myers (2)
+- [ardalis](https://github.com/ardalis) - Steve Smith (2)
+- [bb-froggy](https://github.com/bb-froggy) (2)
+- [dawedawe](https://github.com/dawedawe) (2)
+- [DSivtsov](https://github.com/DSivtsov) - Denis (2)
+- [eNeRGy164](https://github.com/eNeRGy164) - Michaël Hompus (2)
+- [ghogen](https://github.com/ghogen) - Gordon Hogenson (2)
+- [IvenBach](https://github.com/IvenBach) (2)
+- [jmarkman](https://github.com/jmarkman) - Jonathan Markman (2)
+- [jurre](https://github.com/jurre) - Jurre (2)
+- [kthy](https://github.com/kthy) - Kristian Thy (2)
+- [MahmodAbdou](https://github.com/MahmodAbdou) (2)
+- [Marusyk](https://github.com/Marusyk) - Roman Marusyk (2)
+- [MirkoCovizzi](https://github.com/MirkoCovizzi) - Mirko Covizzi (2)
+- [nnpcYvIVl](https://github.com/nnpcYvIVl) (2)
+- [oboukli](https://github.com/oboukli) - Omar Boukli-Hacene (2)
+- [OzBob](https://github.com/OzBob) (2)
+- [shyboylpf](https://github.com/shyboylpf) - Mart (2)
+- [TianqiZhang](https://github.com/TianqiZhang) - Tianqi Zhang (2)
+- [Varorbc](https://github.com/Varorbc) - Varorbc (2)
+- [v-maudel](https://github.com/v-maudel) - Mauricio de los Santos (2)
+- [wadepickett](https://github.com/wadepickett) - Wade Pickett (2)
+- [WindOfMind](https://github.com/WindOfMind) - Dmitriy Aflitonov (2)
+- [dtriana](https://github.com/dtriana) - David Triana (1)
+- [mvelosop](https://github.com/mvelosop) - Miguel Veloso (1)

```

Comment:
vendor

Reply:
I'll manually edit those tonight when I'm back in front of a computer. This tells me that the code needs to be cleaned up to avoid those problems.